### PR TITLE
M sc1 59 19

### PR DESCRIPTION
**Pull request by DrYarosh**
Please check my work.

### CODE REVIEW

File: ayarohovich/task-5-2.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        li ul {
+        list-style-type: lower-roman;
+        }
+        .code {
+		width: 400px;
+		max-height: 4rem;
+        overflow: auto;		
+        background-color: black;
+        color: green;
+        line-height: 1; /* не могу добиться толка с высотой блока*/
```
Comment (pluhin) [inline ayarohovich/task-5-2.html]: line-height: 1rem;
max-height: 4rem;

File: ayarohovich/task-5-3.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                float: left;
+            }
+            .article{
+                
+                margin-left: 15em;
+                                 
+            }
+            .article2 {
+                background-color: cyan;
+                float: left;
```
Comment (pluhin) [inline ayarohovich/task-5-3.html]: float here? Why?

File: ayarohovich/task4-6.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html> 
+<HTML> 
+<HEAD><META charset="utf-8"><TITLE>Пример страницы</TITLE>
+    
+        <!--Не судите строго, я тут накрутил, чтобы посмотреть как будут взаимодествоать между собой теги, стили, классы-->
```
Comment (pluhin) [inline ayarohovich/task4-6.html:5]: Ok :)

**Review by pluhin — CHANGES_REQUESTED** [review]

File: ayarohovich/task-5-8.html [hunk 4]
```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html> 
+<HTML> 
+<HEAD>
+    <META charset="utf-8"><TITLE>Пример страницы</TITLE>
+        <style type="text/css"> /*прошу прощения за такую "красоту"*/
```
Comment (pluhin) [inline ayarohovich/task-5-8.html:5]: Do not worry, at least it works, many who do not even send it

**Review by pluhin — APPROVED** [review]
Good, approved

**Comment by DrYarosh** [comment]
Thank. not enough time to properly arrange. Children, family, work))

пт, 25 янв. 2019 г., 21:21 Siarhei Pishchyk <notifications@github.com>:

**Pull request merged**